## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 429579`

Logs: https://github.com/amadejkastelic/nixpkgs-review-gha/actions/runs/16621842705


---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>superfile</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>superfile</summary>
<pre>Building subPackage ./src/internal/ui/metadata
Building subPackage ./src/internal/ui/prompt
Building subPackage ./src/internal/ui/rendering
Building subPackage ./src/internal/ui/sidebar
Building subPackage ./src/internal/utils
Building subPackage ./src/pkg/file_preview
Building subPackage ./src/pkg/string_function
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
ok  	github.com/yorukot/superfile/src/internal	0.338s
ok  	github.com/yorukot/superfile/src/internal/common	0.005s
--- FAIL: TestGetMetadata (0.00s)
    metadata_test.go:19: 
        	Error Trace:	/build/source/src/internal/ui/metadata/metadata_test.go:19
        	Error:      	Received unexpected error:
        	            	error when executing command: exec: "exiftool": executable file not found in $PATH
        	Test:       	TestGetMetadata
FAIL
FAIL	github.com/yorukot/superfile/src/internal/ui/metadata	0.004s
FAIL</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>superfile</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>superfile</summary>
<pre>Building subPackage ./src/internal/ui/metadata
Building subPackage ./src/internal/ui/prompt
Building subPackage ./src/internal/ui/rendering
Building subPackage ./src/internal/ui/sidebar
Building subPackage ./src/internal/utils
Building subPackage ./src/pkg/file_preview
Building subPackage ./src/pkg/string_function
Running phase: checkPhase
@nix { "action": "setPhase", "phase": "checkPhase" }
ok  	github.com/yorukot/superfile/src/internal	0.263s
ok  	github.com/yorukot/superfile/src/internal/common	0.004s
--- FAIL: TestGetMetadata (0.00s)
    metadata_test.go:19: 
        	Error Trace:	/build/source/src/internal/ui/metadata/metadata_test.go:19
        	Error:      	Received unexpected error:
        	            	error when executing command: exec: "exiftool": executable file not found in $PATH
        	Test:       	TestGetMetadata
FAIL
FAIL	github.com/yorukot/superfile/src/internal/ui/metadata	0.003s
FAIL</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>superfile</li>
  </ul>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:white_check_mark: 1 package built:</summary>
  <ul>
    <li>superfile</li>
  </ul>
</details>

--- a/pkgs/by-name/su/superfile/package.nix
+++ b/pkgs/by-name/su/superfile/package.nix
@@ -1,9 +1,11 @@
 {
   lib,
+  stdenv,
   buildGoModule,
   fetchFromGitHub,
   nix-update-script,
   writableTmpDirAsHomeHook,
+  exiftool,
 }:
 let
   version = "1.3.2";
@@ -27,10 +29,19 @@ buildGoModule {
     "-w"
   ];
 
+  nativeBuildInputs = [ exiftool ];
+
   nativeCheckInputs = [ writableTmpDirAsHomeHook ];
 
-  # Upstream notes that this could be flakey, and it consistently fails for me.
-  checkFlags = [ "-skip=^TestReturnDirElement/Sort_by_Date$" ];
+  # Upstream notes that this could be flaky, and it consistently fails for me.
+  checkFlags = [
+    "-skip=^TestReturnDirElement/Sort_by_Date$"
+  ]
+  ++ lib.optionals stdenv.isDarwin [
+    # Only failing on nix darwin. I suspect this is due to the way
+    # darwin handles file permissions.
+    "-skip=^TestCompressSelectedFiles"
+  ];
 
   passthru.updateScript = nix-update-script { };
 

--- a/pkgs/by-name/su/superfile/package.nix
+++ b/pkgs/by-name/su/superfile/package.nix
@@ -8,7 +8,7 @@
   exiftool,
 }:
 let
-  version = "1.3.2";
+  version = "1.3.3";
   tag = "v${version}";
 in
 buildGoModule {
@@ -19,7 +19,7 @@ buildGoModule {
     owner = "yorukot";
     repo = "superfile";
     inherit tag;
-    hash = "sha256-IzdaOJcwi7+8d8QpTLPJwEhffEz4h0Rdv7APOMcnTHw=";
+    hash = "sha256-A1SWsBcPtGNbSReslp5L3Gg4hy3lDSccqGxFpLfVPrk=";
   };
 
   vendorHash = "sha256-sqt0BzJW1nu6gYAhscrXlTAbwIoUY7JAOuzsenHpKEI=";

